docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -398,6 +398,25 @@ public class CustomConfiguration {
 }
 ----
 
+==== End User attributes
+
+When enabled, Quarkus adds OpenTelemetry End User attributes as Span attributes.
+The attributes are only added when authentication has already happened on a best-efforts basis.
```

Comment:
Can we elaborate a little more on what this means in practice?

Reply:
This feature is one that exposes unexpected inner behaviour, from the user perspective. Most people expect when using authentication, that authentication and its data will be available everywhere, which is not the case...

I think it's useful to explain this to the user and that the id and roles will be added in a best effort way.
Meaning that if the span is created in a place where the CDI context is active the data will be added.
With the proactive authentication on, more places will generate those attributes. Mentioning that this happens because of the place where the `SecurityIdentity` is accessed is useful. 

Does the proactive authentication have a performance penalty? If yes, we should warn about it.

Reply:
I think it's good enough for now.